hedera-node/src/test/java/com/hedera/services/txns/crypto/CryptoTransferTransitionLogicTest.java

```diff
@@ -257,17 +255,6 @@ void computesFailureIfImpliedTransfersNotInSpan() {
 		assertEquals(TRANSFERS_NOT_ZERO_SUM_FOR_TOKEN, validity);
 	}
 
-	@Test
-	void translatesUnknownException() {
-		givenValidTxnCtx(withAdjustments(a, -2L, b, 1L, c, 1L));
-
-		// when:
-		subject.doStateTransition();
-
-		// then:
-		verify(txnCtx).setStatus(FAIL_INVALID);
-	}
-
```

Comment:
This test was not well set up. I am ok with removing it now. We will add another test to cover the `FAIL_INVALID` case.